packages/sku/src/utils/polyfillRegistry.ts

```diff
@@ -0,0 +1,42 @@
+export interface PolyfillRegistryEntry {
+  reason: string;
+  docsUrl?: string;
+}
+
+export type PolyfillRegistry = Record<string, PolyfillRegistryEntry>;
+
+/**
+ * Registry of polyfills that are unnecessary for sku's supported browsers
+ */
+export const POLYFILL_REGISTRY: PolyfillRegistry = {
+  'core-js': {
+    reason:
+      'Most ES2015-ES2017 features are natively supported in modern browsers',
+    docsUrl: 'https://github.com/zloirock/core-js#usage',
+  },
+  '@babel/polyfill': {
+    reason: 'Deprecated package, replaced by core-js',
+    docsUrl: 'https://babeljs.io/docs/en/babel-polyfill',
+  },
+  'regenerator-runtime': {
+    reason: 'async/await is natively supported in modern browsers',
```

Comment:
I'd prefer the reason be a bit more precise, rather than simply saying "it's modern". E.g we should mention should mention "[Baseline](https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility#baseline_badges) widely available" or "Baseline 2022". For example:

```suggestion
    reason: 'async/await is well established and works across many devices and browser versions.',
```

(This wording was taken directly from the "Baseline widely available" messaging, but it doesn't have to be an exact copy)